tests/test_domain_limiter.py

```
@@ -0,0 +1,199 @@
+
+from update_jobs import (  # noqa: E402
+    DomainConcurrencyLimiter,
+    fetch_google_jobs_parallel,
+    fetch_greenhouse_jobs,
+    fetch_lever_jobs,
+    fetch_workday_jobs,
+)
+
+
+class _FakeResponse:
+    def __init__(self, payload, status_code=200):
```

Ruff (0.15.2)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)